ui/src/components/filebrowser/FileBrowser.vue

```diff
@@ -48,19 +52,32 @@ const props = defineProps({
   showDownload: Boolean,
 });
 
+const dataset = ref(null);
 const data_loading = ref(false);
 const fileList = ref([]);
 const searchResults = ref([]);
 const files = computed(() => {
   return isInSearchMode.value ? searchResults.value : fileList.value;
 });
 
+const fetch_files = () => {
+  get_file_list(pwd.value);
+};
+
 function get_file_list(path) {
   data_loading.value = true;
-  datasetService
-    .list_files({ id: props.datasetId, basepath: path })
-    .then((res) => {
-      fileList.value = res.data;
+  Promise.all([
```

Comment:
Could you please add a comment here explaining why we are fetching the dataset whenever we are fetching files?

Reply:
https://github.com/IUSCA/bioloop/blob/173-duplicates-new/ui/src/components/filebrowser/FileBrowser.vue#L70